scripts/stability.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const fs = require('fs');
+const packageJsonPath = process.argv[2] || path.join(process.cwd(), 'package.json');
+const package = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
+const libdir = path.join(path.dirname(packageJsonPath), 'lib');
+const libsrc = fs.existsSync(libdir) ? fs.readdirSync(libdir).filter(x => !x.endsWith('.generated.ts') && !x.endsWith('.js') && x !== 'index.ts') : [ 'dummy' ];
```

Comment:
I think that's the point where I feel comments are needed... Is this tool expected to run on a source tree? Customer dependency tree?

Also that'll definitely be invoked with packages that aren't of construct libraries... I think those should show something else than `unknown`, maybe like "not a construct library" or something.

Reply:
Improved coding style.